Rohit/PostgresAssignment/controller/class.ts

```diff
@@ -0,0 +1,53 @@
+import { Context } from "vm";
+
+const cls = require('../sql/class');
+
+interface classesinfo {
```

Comment:
can we use camelcase for all variable functions class
Interface can be prefixed with I, like IClassInfo